src/integration.test.ts

```diff
@@ -220,15 +220,18 @@ describe(`Integration test for portal ${portal}`, () => {
 
       // Get file content and check returned values.
 
-      const { data, contentType, metadata, portalUrl, skylink: returnedSkylink } = await client.getFileContent(skylink);
+      const { data, contentType, portalUrl, skylink: returnedSkylink } = await client.getFileContent(skylink);
       expect(data).toEqual(fileData);
       expect(contentType).toEqual("text/plain");
-      expect(metadata).toEqual(plaintextMetadata);
       expect(portalUrl).toEqualPortalUrl(portal);
       expect(skylink).toEqual(returnedSkylink);
     });
 
     it("Should get plaintext file metadata", async () => {
+      // TODO: Remove once the metadata changes are in prod.
+      const portal = "https://siasky.dev";
```

Comment:
We might need a dedicated server for this. Something like `siasky.edge` which has the upcoming changes to skyd but is also stable enough. Putting `dev` here is super risky.

Reply:
edge.siasky.net sounds like a cool name for a server, I would actually make it so it deploys automatically after each change to skyd master, skynet-webrportal master or accounts master